### PR TITLE
[4.0] Routing not working in com_wrapper

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

Fixes routing not working in com_wrapper.

### Testing Instructions

Create Wrapper -> Iframe Wrapper menu item.
Inspect its link in menu module.

### Expected result

Link with just menu alias, e.g. `wrapper-item`.

### Actual result

Link like this: `wrapper-item?view=wrapper`.

### Documentation Changes Required

No.

### CODE REVIEW

File: administrator/components/com_wrapper/wrapper.xml [hunk 1]
```diff
@@ -12,7 +12,6 @@
 	<description>COM_WRAPPER_XML_DESCRIPTION</description>
 	<namespace>Joomla\Component\Wrapper</namespace>
 	<files folder="site">
-		<filename>router.php</filename>
```
Comment (brianteeman) [inline administrator/components/com_wrapper/wrapper.xml:15]: Dont you need to ADD 
`<folder>Service</folder>`
??

Comment (SharkyKZ) [reply administrator/components/com_wrapper/wrapper.xml:15]: Added.

**Comment by astridx** [comment]
I have tested this Patch successful.

1. git fetch origin pull/27702/head:j4/wrapper-router
2. git checkout j4/wrapper-router
3. **No** new install or npm ci is needed.
4. I created a menu item of type wrapper
Menus  New Item   admin   Administration

5. Without the patch the URL 
`http://localhost/joomla-cms4/index.php/wrapper?view=wrapper`
is used when I click on the menu item in front end.
With the path the used URL is
`http://localhost/joomla-cms4/index.php/wrapper
`
I tested with the current ‎4.0.0-beta1-dev on Ubuntu with PHP 7.2 and MySQL 5.7.

**Comment by astridx** [comment]
I have tested this item :white_check_mark: successfully on 2ebef575d43ba910a9e9c287bf1c889cf2d33643<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27702">issues.joomla.org/tracker/joomla-cms/27702</a>.</sub>

**Comment by astridx** [comment]
I have tested this item :white_check_mark: successfully on 2ebef575d43ba910a9e9c287bf1c889cf2d33643<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27702">issues.joomla.org/tracker/joomla-cms/27702</a>.</sub>

**Comment by astridx** [comment]
I have tested this item :white_check_mark: successfully on 2ebef575d43ba910a9e9c287bf1c889cf2d33643<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27702">issues.joomla.org/tracker/joomla-cms/27702</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 2ebef575d43ba910a9e9c287bf1c889cf2d33643<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27702">issues.joomla.org/tracker/joomla-cms/27702</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27702">issues.joomla.org/tracker/joomla-cms/27702</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**